_configs/eval_ssm.yaml

```diff
@@ -5,12 +5,38 @@ spreading_model:
     p_values: [0.25, 0.5, 0.75]
 
 networks: [
+  # "artificial_er",
```

Comment:
(1) I suggest to uncomment all test networks that are here
https://github.com/network-science-lab/infmax-trainer-icm-mln/blob/master/_configs/ssnet.yaml

Reply:
I'm sorry, but that would take too much time